content/SCALE/SCALETutorials/DataProtection/ScrubTasksSCALE.md

```diff
@@ -15,37 +15,51 @@ keywords:
 When TrueNAS performs a scrub, ZFS scans the data on a pool.
 Scrubs identify data integrity problems, detect silent data corruptions caused by transient hardware issues, and provide early disk failure alerts.
 
-## Default Scrub Tasks
-TrueNAS generates a default scrub task when you create a new pool and sets it to run every Sunday at 12:00 AM.
+{{< hint type=note >}}
+Starting in TrueNAS 25.10, scrub tasks are managed directly from each pool's settings on the Storage Dashboard instead of as separate scheduled tasks. This article covers the integrated scrub management approach.
```

Comment:
do not use possessive case!
```suggestion
Starting in TrueNAS 25.10, scrub tasks are managed directly from each pool settings on the **Storage Dashboard** instead of as separate scheduled tasks. This article covers the integrated scrub management approach.
```